apps/charterafrica/src/components/Layout/Layout.test.js

```diff
@@ -15,6 +15,14 @@ const defaultProps = {
       src: "/images/eu.png",
     },
   },
+  navbar: {
+    logo: {
+      alt: "Charter Africa",
+      src: "/images/charter-logo.svg",
+      width: "145",
+      height: "40",
+    },
+  },
```

Comment:
Where/how does `Layout` use this?

Reply:
For some reason, the test fails if not included. I can see in the generated snapshot that it is used.

Reply:
@kilemensi I kind of can't figure this out, if the `Navbar` can't be placed in the `Layout`, where should it be placed?

Reply:
Not sure I follow @kelvinkipruto. Are you having any issues adding `NavBar` in the `Layout`?

**My understanding**
When I raised the issue _(3 days ago)_, there was no `NavBar` but `navBar` prop was still being set in `Layout` and hence me asking what is it doing there?

You added `NavBar` to `Layout` _yesterday_ and hence me saying
>You've now added NavBar to the Layout so of course tests would fail. **_Previously_** there was no need for it.

i.e. If we `NavBar`, then having `navBar` prop justified.

Reply:
Understood. This is then now resolved